packages/contracts/tsconfig.json

```diff
@@ -3,6 +3,6 @@
   "compilerOptions": {
     "resolveJsonModule": true
   },
-  "include": ["./test", "src/**/*", "deployments"],
+  "include": ["./test", "src/**/*", "deployments", "deploy", "bin"],
```

Comment:
Any idea why tsconfig needs to change for this?

Reply:
See https://github.com/ethereum-optimism/optimism/pull/1005#discussion_r644470227